.github/workflows/ci.yml

```diff
@@ -5,22 +5,16 @@ on: [ push, pull_request ]
 jobs:
   build:
     runs-on: ubuntu-latest
-    defaults:
```

Comment:
Any reason why we're abandoning the environment settings? I don't have a strong preference either way, but was curious.

Reply:
Huh, I think this snuck in like the approx change below. I'll revert this!